install_config/master_node_configuration.adoc

```diff
@@ -1203,8 +1203,21 @@ openshift_master_audit_config={"enabled": true, "auditFilePath": "/var/log/openp
 
 The advanced audit feature provides several improvements over the
 xref:master-node-config-audit-config[basic audit functionality], including
-fine-grained events filtering and multiple output back ends. The following table
-contains additional options you can use.
+fine-grained events filtering and multiple output back ends. 
+
+To enable the advanced audit feature, provide the following values in the `openshift_master_audit_config` parameter
+
+----
+openshift_master_audit_config={"enabled": true, "auditFilePath": "/var/log/oscp-audit/-oscp-audit.log", "maximumFileRetentionDays": 14, "maximumFileSizeMegabytes": 500, "maximumRetainedFiles": 5, "policyFile": "/etc/security/adv-audit.yaml","logFormat":"json"}
```

Comment:
Nitpick: missing space before comma in `"policyFile": "/etc/security/adv-audit.yaml","logFormat":"json"`

Reply:
@git001, will you update the PR and squash your commits? (If you don't, I'll open up a follow-up PR to do so.) Thanks!